Cargo.lock

```diff
@@ -438,7 +440,7 @@ dependencies = [
 
 [[package]]
 name = "serde_json"
-version = "1.0.45"
+version = "1.0.48"
```

Comment:
Is this version bump necessary? If not, please don't do it. I try to make sure to update all dependencies locally so that I can audit changes carefully.

Reply:
Sorry, I added the latest `serde_json` because it was a new dev dependency for this crate, I can reduce its minimum version req and undo the lockfile changes.